app/src/main/java/com/smlnskgmail/jaman/hashchecker/features/settings/view/lists/weblinks/WebLinksListAdapter.java

```diff
@@ -21,10 +21,10 @@ public class WebLinksListAdapter extends BaseListAdapter<WebLink> {
     private final Context context;
 
     WebLinksListAdapter(
+            Context context,
```

Comment:
You also should add `@NonNull` annotation to `Context`

Reply:
Done